src/domain/entities/atlassian-design.ts

```diff
@@ -42,5 +43,7 @@ export type AtlassianDesign = {
 	readonly status: AtlassianDesignStatus;
 	readonly type: AtlassianDesignType;
 	readonly lastUpdated: string;
+	readonly lastUpdatedBy?: FigmaUser;
```

Comment:
**question:** From what I see in the `Design` type declaration in Atlassian's codebase, `lastUpdatedBy` is of the following type (converted to TS for convenience):

```ts
interface User {
  accountId: string|null;
}
```

Taking this into account, please, could you double-check that the data the app ingests matches the expected contract? From what I see:
- `FigmaUser` does not have `accountId`.
- `FigmaUser` has an extra `email` field with sensitive personal information. Even if sending an extra field does not break anything, it is better to avoid sending this information among services.

If there is mismatch, consider adding another type `AtlassianProviderUser` before `AtlassianDesign` (by analogue with `AtlassianDesignStatus`) as it is _specific_ to `AtlassianDesign`. Then, we could transform `FigmaUser` to `AtlassianProviderUser` in transformers (e.g., `transformFileToAtlassianDesign`).

Please, let me know if you have any concerns.

Reply:
Good call, the aim is to map the external id in FigmaUser to accountId during ingestion.
Having `AtlassianProviderUser` and transforming it by stripping the email and only including the external id is the better approach. Will update it shortly.